core/src/main/java/io/questdb/cairo/ColumnType.java

```diff
@@ -29,10 +29,20 @@
 
 // ColumnType layout - 32bit
 //
-// | PGWire format | Extra type information | Type discriminant (tag) |
+// | Handling bit  | Extra type information | Type discriminant (tag) |
 // +---------------+------------------------+-------------------------+
 // |    1 bit      |        23 bits         |         8 bits          |
 // +---------------+------------------------+-------------------------+
+//
+// Handling bit:
+//   Skip column use case:
```

Comment:
Alex mentioned we still uses negative numbers in the column type in Java in the wal apply logic.

Reply:
In the code base, negative values may mean deleted column/column to be deleted, not only a column to be skipped. In that case, all other bits aren't used. Let's keep the description as is, but I find it slightly confusing since negative values aren't valid column types, but special sentinel values.